tests/pipeline_backend_test.py

```diff
@@ -34,8 +34,8 @@ def test_filter_by_key_must_not_be_none(self):
             col = p | "Create PCollection" >> beam.Create(data)
             key_to_keep = None
             with self.assertRaises(TypeError):
-                result = self.ops.filter_by_key(col, key_to_keep,
-                                                "filte_by_key")
+                result = self.backend.filter_by_key(col, key_to_keep,
+                                                    "filte_by_key")
```

Comment:
nit: "filter_by_key". Also in the rest of this file.

Reply:
Thanks!